frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -29,51 +29,35 @@ export type PatientIdsTriplet = {
   DMP_ID: string | null;
 };
 
-function patientAliasFilterWhereVariables(
+function patientFilterWhereVariables(
   parsedSearchVals: string[]
-): PatientAliasWhere[] {
+): PatientWhere[] {
   if (parsedSearchVals.length > 1) {
     return [
-      { value_IN: parsedSearchVals },
-      { namespace_IN: parsedSearchVals },
       {
-        isAliasPatients_SOME: {
-          hasSampleSamples_SOME: {
-            hasMetadataSampleMetadata_SOME: {
-              cmoSampleName_IN: parsedSearchVals,
-            },
-          },
+        patientAliasesIsAlias_SOME: {
+          value_IN: parsedSearchVals,
         },
       },
       {
-        isAliasPatients_SOME: {
-          hasSampleSamples_SOME: {
-            hasMetadataSampleMetadata_SOME: {
-              primaryId_IN: parsedSearchVals,
-            },
+        hasSampleSamples_SOME: {
+          hasMetadataSampleMetadata_SOME: {
+            cmoSampleName_IN: parsedSearchVals,
           },
         },
       },
     ];
   } else {
     return [
-      { value_CONTAINS: parsedSearchVals[0] },
-      { namespace_CONTAINS: parsedSearchVals[0] },
       {
-        isAliasPatients_SOME: {
-          hasSampleSamples_SOME: {
-            hasMetadataSampleMetadata_SOME: {
-              cmoSampleName_CONTAINS: parsedSearchVals[0],
-            },
-          },
+        patientAliasesIsAlias_SOME: {
+          value_CONTAINS: parsedSearchVals[0],
         },
       },
       {
-        isAliasPatients_SOME: {
-          hasSampleSamples_SOME: {
-            hasMetadataSampleMetadata_SOME: {
-              primaryId_CONTAINS: parsedSearchVals[0],
-            },
+        hasSampleSamples_SOME: {
+          hasMetadataSampleMetadata_SOME: {
+            cmoSampleName_CONTAINS: parsedSearchVals[0],
           },
```

Comment:
This function gets called when the user performs a search. Since we're querying by `Patients` now instead of `PatientAlias`, I'm updating where GraphQL searches for data.